src/action-menu/ActionMenu.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import ContentContainer from 'terra-content-container';
+import Header from './ActionMenuHeader';
+import {
+  generateOnKeyDown,
+  itemByDirection,
+  itemByChar,
+  flattenActionItems,
+} from './_ActionUtils';
+
+import styles from './ActionMenu.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * Label for the ActionMenu.
+   */
+  ariaLabel: PropTypes.string.isRequired,
+  /**
+   * The child ActionMenu elements.
+   */
+  children: PropTypes.node,
+  /**
+   * Whether or not a header should be displayed using the ariaLabel.
+   */
+  isHeaderDisplayed: PropTypes.bool,
+  /**
+   * The close callback.
+   */
+  onRequestClose: PropTypes.func,
```

Comment:
Does this callback receive any parameters?

Reply:
Update.